airflow/providers/jenkins/sensors/jenkins_build.py

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
The file name should be just `jenkins.py`
So the import will be:

`from airflow.providers.jenkins.sensors.jenkins import JenkinsBuildSensor`

This is the convention (unless special circumstances) for example you can see:
`from airflow.providers.http.sensors.http import HttpSensor`

Reply:
changed the file name to `jenkins.py`